src/jvm/test/org/jetbrains/kotlinx/lincheck_test/AbstractLincheckTest.kt

```diff
@@ -68,4 +68,4 @@ abstract class AbstractLincheckTest(
     }
 }
 
-private const val TIMEOUT = 3 * 60_000L // 3 min
\ No newline at end of file
+private const val TIMEOUT = 10 * 60_000L // 10 min
```

Comment:
In the most of the builds (in teamcity and on my machine) the performance drop was insignificant. Take a look on the build times in this PR and, for example, this one: https://github.com/JetBrains/lincheck/pull/333). The reason why I increased the timeout is that the only build where we have a considerable build time increase is "Model checking tests on a single CPU". Now this build takes 13 minutes, while previously it took 9 minutes. But it's completely expected, because the tests where time increased are "SpinLockSetTest" and "ReentrantLockSetTest". Summarising: this timeout increase was for only two tests in the only one build.

Reply:
Thanks for the explanation! I agree that this performance degradation is insignificant.